modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/MultiStepNav.es.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayMultiStepNav from '@clayui/multi-step-nav';
+import React, {useState} from 'react';
+
+import Sheet from './Sheet.es';
```

Comment:
```suggestion
import Sheet from './Sheet';
```

https://github.com/pt-liferay-solutions/liferay-portal/pull/2143#pullrequestreview-1106273108